.drone/drone.jsonnet

```diff
@@ -447,6 +447,9 @@ local manifest_ecr(apps, archs) = pipeline('manifest-ecr') {
 [
   pipeline('loki-build-image') {
     local build_image_tag = '0.25.0',
+    node: {
+      type: 'no-parallel',
```

Comment:
Is this something we want? @bboreham could you link to the discussion you had with Julien (referenced [here](https://github.com/grafana/loki/pull/8047#discussion_r1064104928))?

Reply:
Ideally you don't want it.

Reply:
Thank you. Reading the thread, I'm not convinced this is the right approach (at least in isolation); we have some more fundamental issues with the way our CI runs which needs attention.

Any objections to reverting this change?

Reply:
I filed #8076 for the broader CI issue.

Reply:
Many thanks! @ashwanthgoli please remove this change and re-generate the drone signature.